CONTRIBUTING.md

```diff
@@ -0,0 +1,8 @@
+# Adding a new modeling command
+
+ - In `each_command.rs` add your new `struct MyNewCommand`
```

Comment:
It might be helpful to note that this struct should contain the inputs of the command, though that may be implied?